src/ucp/wireup/wireup.c

```diff
@@ -769,9 +792,10 @@ ucp_wireup_connect_lane_to_iface(ucp_ep_h ep, ucp_lane_index_t lane,
         if ((wireup_ep == NULL) && !ucp_ep_has_cm_lane(ep)) {
             /* Lane already exists */
             return UCS_ERR_UNREACHABLE;
-        } else if (((wireup_ep == NULL) ||
-                    (wireup_ep->super.uct_ep != NULL))) {
-            /* UCT EP has already been allocated for this EP */
+        } else if ((wireup_ep == NULL) ||
+                   (wireup_ep->super.uct_ep != NULL)) {
+            /* UCT EP has already been allocated for this EP (it could be
+             * done only during CM connection establishment phase) */
```

Comment:
shouldn't the reconfig logic already make sure that we create only "new" endpoints, that weren't already created in the previous stage?
same comment for ucp_wireup_connect_lane_to_ep

Reply:
is it ok to pass reuse_lane_map to `connect_lane_to_[ep, iface]`? they could allow us to check which EPs were created

Reply:
we should not call ucp_wireup_connect_lane_to_iface for reused ep's in the first place
so maybe return lane_map to ucp_wireup_init_lanes() function, from ucp_wireup_check_config_intersect